packages/governance-modules/test/contracts/modules/ElectionModule/DebtShare.test.js

```diff
@@ -0,0 +1,195 @@
+const { ethers } = hre;
```

Comment:
This new tests basically checks that a snapshot is taken on the first nomination of each epoch. Nothing else for now. Wondering if we should also check vote power stuff?